configs/NBD_HUMMINGBIRD200RS/config.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #pragma once
+
+ #define FC_TARGET_MCU     AT32F435G
+ 
+ #define BOARD_NAME        HUMMINGBIRD_200_RS
+ #define MANUFACTURER_ID   NEBD
+ 
+ #define USE_ACC
+ #define USE_ACC_SPI_ICM42688P
+ #define USE_GYRO
+ #define USE_GYRO_SPI_ICM42688P
+ 
+ #define USE_BARO
+ #define USE_BARO_DPS310
+ #define USE_MAG
+ #define USE_MAG_QMC5883
+ #define USE_FLASH
+ #define USE_FLASH_M25P16
+ #define USE_MAX7456
+ 
+ #define BEEPER_PIN           PB0
+ #define MOTOR1_PIN           PA8
+ #define MOTOR2_PIN           PB10
+ #define MOTOR3_PIN           PA0
+ #define MOTOR4_PIN           PA1
+ #define LED_STRIP_PIN        PB1
+ #define UART1_TX_PIN         PA9
+ #define UART2_TX_PIN         PA2
+ #define UART6_TX_PIN         PA4
+ #define UART1_RX_PIN         PA10
+ #define UART2_RX_PIN         PA3
+ #define UART3_RX_PIN         PB11
+ #define UART6_RX_PIN         PA5
+ #define LED0_PIN             PC14
```

Comment:
PC13, P14 or PC15 are not to be used with leds.

See: https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines#3146-leds